constants.go

```diff
@@ -18,10 +18,14 @@ package minio
 
 /// Multipart upload defaults.
 
-// miniPartSize - minimum part size 64MiB per object after which
+// minPartSize - minimum part size 64MiB per object after which
 // putObject behaves internally as multipart.
 const minPartSize = 1024 * 1024 * 64
 
+// copyPartSize - default (and maximum) part size to copy in a
+// copy-object request.
+const copyPartSize = 1024 * 1024 * 1024 * 5
```

Comment:
This number is actually 5 GiB. I will update the comment to make this more apparent.

Since this is only server-side copying, the most efficient thing to do is to reduce the number of network requests - this is done by copying the largest possible object-part in each request.